ddtrace/contrib/algoliasearch/patch.py

```diff
@@ -0,0 +1,138 @@
+from importlib import import_module
+
+from ddtrace.compat import stringify
+from ddtrace.ext import AppTypes
+from ddtrace.pin import Pin
+from ddtrace.settings import config
+from ddtrace.utils.wrappers import unwrap as _u
+from ddtrace.vendor.wrapt import wrap_function_wrapper as _w
+
+DD_PATCH_ATTR = '_datadog_patch'
+
+SERVICE_NAME = 'algoliasearch'
+APP_NAME = 'algoliasearch'
+SEARCH_SPAN_TYPE = 'algoliasearch.search'
+
+# Default configuration
+config._add('algoliasearch', dict(
+    service_name=SERVICE_NAME,
+    app=APP_NAME,
```

Comment:
You don't need to add `app` and `app_type` to the config.

Basically anything we add to the config is something we want users to be able to override. We want `app` and `app_type` to be static for this integration.

Reply:
Ok, will remove. Btw, flask allows those to be overridden.

Reply:
haha, do as I say not as I do? 😆